tests/pages/jobs/JobDetails-cy.js

```diff
@@ -84,9 +70,7 @@ xdescribe('Job Details', function () {
 
       cy.get('.page table .expanding-table-child')
         .should(function ($children) {
-          // Four table columns, two table rows, each with two children.
-          // 4 * 2 * 2 = 16
-          expect($children.length).to.equal(16);
+          expect($children.length).to.equal(20);
```

Comment:
Why did we remove the explanatory text and did the number of children just change? When I count it I still get 16. What changed? What is it counting that it maybe shouldn't?
I would like a little explanation here